### PR TITLE
mchmarny after migration cleanup

### PR DESCRIPTION
**Pull request by mchmarny**
Updates the repo path in all the non-go files (release, test, readme etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +68,7 @@ apk add ko
 With Go 1.18+, build and install the latest released version:
 
 ```
-go install github.com/google/ko@latest
+go install github.com/ko-build/ko@latest
```
Comment (imjasonh) [inline README.md:71]: This one will actually have to wait until we rename the Go import path. Even though the repo name has changed, as long as the GH redirect is in place, Go considers this `github.com/google/ko`.

File: .ko.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 baseImageOverrides:
-  github.com/google/ko: golang:1.18
+  github.com/ko-build/ko: golang:1.18
```
Comment (imjasonh) [inline .ko.yaml:2]:
```suggestion
  github.com/google/ko: golang:1.18
```

File: .goreleaser.yml [hunk 3]
```diff
@@ -13,7 +13,7 @@ builds:
   flags:
   - -trimpath
   ldflags:
-  - "-s -w -X github.com/google/ko/pkg/commands.Version={{.Version}}"
+  - "-s -w -X github.com/ko-build/ko/pkg/commands.Version={{.Version}}"
```
Comment (imjasonh) [inline .goreleaser.yml:16]:
```suggestion
  - "-s -w -X github.com/google/ko/pkg/commands.Version={{.Version}}"
```

File: integration_test.sh [hunk 4]
```diff
@@ -17,13 +17,13 @@ export GOPATH
...
-mkdir -p "$GOPATH/src/github.com/google/ko"
-cp -r "$ROOT_DIR/"* "$GOPATH/src/github.com/google/ko/"
+mkdir -p "$GOPATH/src/github.com/ko-build/ko"
+cp -r "$ROOT_DIR/"* "$GOPATH/src/github.com/ko-build/ko/"
 
 echo "Downloading github.com/go-training/helloworld"
 GO111MODULE=off go get -d github.com/go-training/helloworld
 
-pushd "$GOPATH/src/github.com/google/ko" || exit 1
+pushd "$GOPATH/src/github.com/ko-build/ko" || exit 1
```
Comment (imjasonh) [inline integration_test.sh:26]: This should refer to google/ko too for now.

**Review by imjasonh — COMMENTED** [review]
Thanks! Some of these will need to wait for the next step.

File: README.md (hunk 1 above)
Comment (BenTheElder) [reply README.md:71]: I couldn't find any discussion, is there one about doing this anywhere (renaming the go import paths)? Or should someone start one and / or help PR that change?

Comment (imjasonh) [reply README.md:71]: This is roughly sketched out in https://github.com/ko-build/ko/issues/791 

I don't have a set timeline for it yet, I think a couple other things have to happen first. I'd guess some time after Kubecon we'll deprecate `github.com/google/ko` and point folks to importing `ko.build/pkg/build` and `go install ko.build@latest`.

As part of that we'll also move the ko.build website into this repo (#749) which also sets the vanity importpath meta tags.

**Pull request closed without merging**